tsconfig.json

```diff
@@ -7,8 +7,9 @@
     "downlevelIteration": true,
     "preserveWatchOutput": true,
     "resolveJsonModule": true,
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "outDir": "build"
   },
   "include": ["**/*.ts"],
-  "exclude": ["node_modules", "transforms/__testfixtures__/**"]
+  "exclude": ["node_modules", "build", "transforms/__testfixtures__/**", "__test__", "**/*.spec.ts"]
```

Comment:
@bjohansebas why is that `build` entry necessary? That's the `"outDir"` for TS config

Reply:
I have added a change to `jest.config`. Please check if it helps with your issue or share more details

Reply:
It always generated this error for me

![image](https://github.com/user-attachments/assets/29f97f8c-b6e9-425d-8f28-3bcbdd9da247)

Reply:
This seems more like an error in our configuration. Maybe we should move these files to the `src` folder?

Reply:
ok, I see it's after running `npm run build` command. It's all good with this `exclude: ["build"]`.

Another workaround could be to update `build` command to be like this:

`rm -rf ./build && tsc -d -p tsconfig.json`

but let's leave ignore in the tsconfig.

We can mere if everything is fine now